ports/libsndfile/vcpkg.json

```
@@ -1,20 +1,50 @@
 {
   "name": "libsndfile",
-  "version-string": "1.0.31",
-  "description": "Library to read, write and manipulate many soundfile types. Authored by Eric de Castro Lopo",
+  "version-semver": "1.1.0",
```

@evpobr, Could you please confirm whether the library officially adheres to semantic versioning or not (breaking changes -> bump major version, non-breaking changes -> bump minor version, non-breaking bug fixes -> bump patch version)?

---

doesn't matter `version-semver` shouldn't be used. Use version instead.
```suggestion
  "version": "1.1.0",
```

---

I didn't understand. so should I or not use `version-semver`?

https://vcpkg.readthedocs.io/en/latest/specifications/versioning/#2-specifying-package-versions

It says here that it is possible.

---

And yes, we use SemVer:

https://github.com/libsndfile/libsndfile/blob/master/CHANGELOG.md

---

''The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)." Thanks for your clarification

---

> I didn't understand. so should I or not use `version-semver`?
> 
> https://vcpkg.readthedocs.io/en/latest/specifications/versioning/#2-specifying-package-versions
> 
> It says here that it is possible.

We are cautious about using "version-semver", if Upstream explicitly says adheres to Semantic Versioning we allow to use “version-semver”